doc/syscalls/70006_lsm303dlhc.md

```diff
@@ -0,0 +1,129 @@
+---
+driver number: 0x70006
+---
+
+# LSM303DLHC
+
+## Overview
+
+Three axis accelerometer, magnetometer and temperature sensor.
+
+[Manual](https://www.st.com/resource/en/datasheet/lsm303dlhc.pdf)
+
+## Command
+
+  * ### Command number: `0`
+
+    **Description**: Returns SUCCESS
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: 0
+
+  * ### Command number: `1`
+
+    **Description**: Verifies whether the hardware sensor is present.
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
```

Comment:
If present is async, it writes and reads from the device. The subscribed callback will return whether the device is present or not. The command just issues the writing and the reading. It can only do this if there is no other command in progress, thus the EBUSY.

Reply:
Ahh.. that makes sense -- my concern then I think is if someone is reading this it would be very easy to think that a return of `SUCCESS` means the hardware is present; I gave suggested updated language.

Reply:
```suggestion
    **Returns**: `SUCCESS` if presence test has started successfully, `EBUSY` otherwise.
```